Documentation/MenuObjects/Tmenu/Index.rst

```diff
@@ -125,24 +360,342 @@ TMENU
          See example in the extension statictemplates
          (statictemplates/media/scripts/example\_itemArrayProcFunc.php).
 
+Item States
+-----------
 
 .. container:: table-row
 
    Property
-         [Common Item States, see above]
+         [Item States, like NO, ACT, CUR etc]
 
    Data type
          ->TMENUITEM
 
    Description
-         This is the TMENUITEM-options for each category of menu item that can
+         This is the TMENUITEM-options for each category of a menu item that can
          be generated.
 
          **Special:**
 
          The ->OptionSplit function is run on the whole configuration before
          the items are generated.
 
+sectionIndex
+------------
+
+.. container:: table-row
+
+   Property
+         sectionIndex
+
+   Data type
+         *(array of keys)*
+
+   Description
+         (see below)
```

Comment:
"see below" is not correct as it is by now a separate page. So either include it into this page or link it accordingly to the other page.